plugins/plugin.go

```diff
@@ -59,6 +63,8 @@ func New(options ...Option) (*Plugin, error) {
 		MetricsHost:       "localhost",
 		MetricsPort:       "61001",
 		ConfigPath:        "",
+		IAMConfigPath:     "",
+		CACertificatePath: "",
```

Comment:
same here, no need to initialize empty string. Or maybe i am not quite understand why are you doing it?